services/backend/src/service/protected_area.js

```diff
@@ -384,28 +384,6 @@ module.exports = (paPersistence, seService) => {
       }
       return {};
     },
-
-    /**
```

Comment:
function `getAreaByCoverage` should also be deleted

Reply:
Done!